README.md

```diff
@@ -233,6 +224,20 @@ This is the directory where your bot saves and retrieves the `session.data` file
 *Please note that as docker is intended for production,
 there is not the possibility for the `DEBUG` mode when using docker.*
 
+### [Vote delegation infrastructure](#vote-delegation)
+If you want to self host infrastructure for vote delegation - unfortunately you need to set it up 
+on your own as this feature is still in beta.
+
+To make it work you need to:
+- deploy [website](https://github.com/block42-blockchain-company/terra-telegram-bot-website)
+- deploy [backend](https://github.com/block42-blockchain-company/terra-telegram-bot-backend) and connect your own wallet
+- setup your own MongoDB database and connect with backend instance
+- set correct endpoints in `constants.py`
+
+If you don't do it you can still use all the features provided by the website using the instance 
+deployed by block42. Unfortunately, you can't use the backend instance (delegate the voting) as it
```

Comment:
you can't use "our" instance. 

He can still use his own right?

Reply:
He can vote with station extension here https://terra-bot.b42.tech/vote using our instance but he can't delegate the vote (https://terra-bot.b42.tech/api). I changed to make it more clear